src/index.js

```diff
@@ -0,0 +1,6 @@
+import './style.css';
+import renderScoreList from './utils/list.js';
+import renderForm from './utils/form.js';
+
```

Comment:
- Kindly fix your linter in your pull reques.

Reply:
Hello Carlos, 
This linter is not something i 've changed, i'm using the same files provided by microverse. 
Do you have any ideas on why the Lighthouse is failing? Have you s